### PR TITLE
Translations into V4

### PR DESCRIPTION
**Pull request by Soapy7261**
WIP

### CODE REVIEW

**Comment by Soapy7261** [comment]
yay pylint doesn't hate me anymore!

**Comment by JoshuaSlui** [comment]
Do you have an ETA on this pull request @Soapy7261?

**Comment by Soapy7261** [comment]
I mean i kinda want your feedback on what to add, since it *could* already be added but it probably shouldn't

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]
This seems fine to me. I haven't tested it yet and i don't see the need to because the code is clearly functional. Though I'd like to ask that you **add indentation between each respective function and test**, because this looks a bit awkward otherwise.

I will most likely mend the code after it is pushed to work with user/guild database settings, but that should not take much work with an understanding of Rebel's systems. You can mark this as ready for review after changing the indentation if you wish, and then i will pull it.

**Comment by Soapy7261** [comment]
That should be what you want, right?

File: providers/translation/get_translate.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        translation_map = {}
+        for dirname, _, filenames in walk("./translations/"):
+            if dirname == "./translations/":
+                continue
+            filename = filenames[0] # There should be only 1
+            with open (f"{dirname}/{filename}", "r", encoding="UTF-8") as f:
+                json_data = load(f)
+                translation_map[dirname.removeprefix("./translations/")] = json_data
+        return translation_map
+    def get_translation(self, language: str, message_code: str) -> str:
```
Comment (JoshuaSlui) [inline providers/translation/get_translate.py]: Missing empty line

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**